questions/List-manipulate.md

```diff
@@ -0,0 +1,58 @@
+# RANDLIST
+
+## Question
+
+Welcome to **RANDLIST**! Here is the problem statement:
+
+You must input a number to determine the length of a list that will contain random numbers from 1-100. The last element of the list should be removed 3 times and you must return the list to be displayed in highest value to lowest value order. 
```

Comment:
Could be a bit more clear? Something like the remove the last element of the list 3 times and then sort the remaining elements